tools/tf2openapi/types/tfsavedmodel.go

```diff
@@ -15,28 +15,23 @@ import (
 const ServingMetaGraphTag string = "serve"
 
 type TFSavedModel struct {
-	MetaGraphs [] TFMetaGraph
```

Comment:
same here

Reply:
changed back to slice so that the internal model fully captures the impt fields of the savedmodel